### DISCUSSION TITLE
Contributing - I think I need to be an official "Contributor"

### DISCUSSION THREAD

**Discussion by sum1els-com** [jacobsalmela/ascii-planets#27] [General]
The instructions suggest simply pushing changes, but I've tried to push a new working branch with some changes and I get permission denied.  You may need to change the settings of the project to allow others to push, or you may need to specifically allocate contributor rights to individuals.  Sorry I don't have much experience with GitHub project management, so I don't know any details.

**Comment by jacobsalmela** [comment]
Try forking and then making a PR.  It should work

**Comment by sum1els-com** [comment]
Normally, such changes would be done in a branch, rather than a fork, which requires write access to the repo.

According to https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork I need to have write access to the repo in order to create a PR anyway.

**Comment by sum1els-com** [comment]
Well, I was able to create a PR for a branch I made in my own fork after all!  https://github.com/jacobsalmela/ascii-planets/pull/28/files
